lib/cubit/payments/payments_cubit.dart

```diff
@@ -50,6 +51,22 @@ class PaymentsCubit extends Cubit<PaymentsState> with HydratedMixin<PaymentsStat
     ).distinct().listen((PaymentsState newState) => emit(newState));
   }
 
+  StreamSubscription<Payment> trackPaymentEvents({
+    required bool Function(Payment) paymentFilter,
+    void Function(Payment)? onData,
```

Comment:
nit: maybe this could be required, there is no point tracking otherwise?

Reply:
True, done: 54d3a970eb5ae45bbb5f1903779229de11f1f5e9